Gemfile.lock

```diff
@@ -18,6 +18,7 @@ GEM
     ast (2.4.2)
     base64 (0.2.0)
     bigdecimal (3.1.6)
+    csv (3.2.8)
```

Comment:
Add csv gem to Gemfile because it is changed from default gem to bundled gem in ruby 3.4.